includes/Core/CLI/CLI_Command.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Site Kit CLI Command
+ *
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\CLI;
+
+use Google\Site_Kit\Context;
+use WP_CLI_Command;
+
+/**
+ * Base CLI Command class.
+ *
+ * @since n.e.x.t
+ */
+class CLI_Command extends WP_CLI_Command {
```

Comment:
I like the idea of using a base command for all CLI commands, I'm just not sure if we want to include the constructor in it. I realize that might make this an empty class for now, but I'm curious what @felixarntz thinks about it.

Reply:
@aaemnnosttv can you elaborate why you are not sure about having the constructor in the parent class?

Reply:
Curious to hear as well, generally +1 for introducing a base class either way.

Reply:
I was thinking about dependencies. Some commands will likely have different dependencies but I suppose they will all require `Context` as the first so that shouldn't really be a problem then 👍